### ISSUE TITLE
Build fails on iOS

### ISSUE THREAD

**Issue by JohnMusleh** [deepak786/intercom_flutter#44]
I created a hello world app with flutter, imported this package and called Intercom.initialize(),
build fails on iOS

I also tried cloning this repository and running the example, same thing, build fails on iOS

> Failed to build iOS app
> Error output from Xcode build:
> ↳
>     ** BUILD FAILED **
> 
> 
> Xcode's output:
> ↳
>     warning: Capabilities for Runner may not function correctly because its entitlements use a placeholder team ID. To resolve this,
>     select a development team in the build settings editor. (in target 'Runner')
>     /Users/musleh/downloads/flutter/.pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.0.5+2/ios/Classes/IntercomFlutterPlugin.m:1
>     16:23: warning: 'presentMessageComposerWithInitialMessage:' is deprecated: '+[Intercom presentMessageComposerWithInitialMessage]'
>     is deprecated. 'Use +[Intercom presentMessageComposer:initialMessage]' instead. [-Wdeprecated-declarations]
>                 [Intercom presentMessageComposerWithInitialMessage:message];
>                           ^
>     In file included from
>     /Users/musleh/downloads/flutter/.pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.0.5+2/ios/Classes/IntercomFlutterPlugin.m:2
>     :
>     /Users/musleh/Development/projects/test_app/ios/Pods/Intercom/Intercom/Intercom.framework/Versions/A/Headers/Intercom.h:244:83:
>     note: 'presentMessageComposerWithInitialMessage:' has been explicitly marked deprecated here
>     + (void)presentMessageComposerWithInitialMessage:(NSString *)message __attribute((deprecated("'+[Intercom
>     presentMessageComposerWithInitialMessage]' is deprecated. 'Use +[Intercom presentMessageComposer:initialMessage]' instead.")));
>                                                                                       ^
>     /Users/musleh/downloads/flutter/.pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.0.5+2/ios/Classes/IntercomFlutterPlugin.m:1
>     18:23: warning: 'presentMessageComposer' is deprecated: '+[Intercom presentMessageComposer]' is deprecated. 'Use +[Intercom
>     presentMessageComposer:initialMessage]' instead. [-Wdeprecated-declarations]
>                 [Intercom presentMessageComposer];
>                           ^
>     In file included from
>     /Users/musleh/downloads/flutter/.pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.0.5+2/ios/Classes/IntercomFlutterPlugin.m:2
>     :
>     /Users/musleh/Development/projects/test_app/ios/Pods/Intercom/Intercom/Intercom.framework/Versions/A/Headers/Intercom.h:239:45:
>     note: 'presentMessageComposer' has been explicitly marked deprecated here
>     + (void)presentMessageComposer __attribute((deprecated("'+[Intercom presentMessageComposer]' is deprecated. 'Use +[Intercom
>     presentMessageComposer:initialMessage]' instead.")));
>                                                 ^
>     2 warnings generated.
>     Undefined symbols for architecture x86_64:
>       "_UISceneWillEnterForegroundNotification", referenced from:
>           +[ICMPresentationManager observeSceneWillEnterForeground] in Intercom(ICMPresentationManager.o)
>       "___isPlatformVersionAtLeast", referenced from:
>           -[IntercomSDK_IntercomConversationCollectionViewController insetsForAccessoryView] in
>           Intercom(IntercomConversationCollectionViewController.o)
>           +[IntercomSDK_ITBUpload createUploadForMediaAsset:completion:] in Intercom(ITBUpload.o)
>           +[IntercomSDK_ITBUpload processImageDataForUpload:mediaAsset:info:imageUploadDataUTI:completion:] in Intercom(ITBUpload.o)
>           -[IntercomSDK_IIMInputAccessoryView createConstraints] in Intercom(IIMInputAccessoryView.o)
>           -[IntercomSDK_IIMInputButtonContainerView createConstraints] in Intercom(IIMInputButtonContainerView.o)
>           -[IntercomSDK_IIMMediaViewController updateCollectionViewContentInsets] in Intercom(IIMMediaViewController.o)
>           -[IntercomSDK_IIMMediaViewController layoutExpandButton] in Intercom(IIMMediaViewController.o)
>           ...
>     ld: symbol(s) not found for architecture x86_64
>     clang: error: linker command failed with exit code 1 (use -v to see invocation)
>     note: Using new build systemnote: Planning buildnote: Constructing build description

**Comment by Zazo032** [comment]
Could you post some more info? Which version of the plugin are you using? Could you also provide the output of `flutter doctor -v` along with a minimum project or code to reproduce the issue? Which version of macOS/iOS/Xcode are you using?

**Comment by JoeCiou** [comment]
I have the same problem.
try to use `flutter clean`

**Comment by JohnMusleh** [comment]
This issue is from the intercom ios SDK.
I found the issue here: https://stackoverflow.com/questions/57917069/apple-mach-o-linker-error-undefined-symbols-for-architecture-x86-64-uiscene

"On 12-09-2019, Intercom released sdk version 5.5.0, which supports iOS 13.
UISceneWillEnterForegroundNotification and isPlatformVersionAtLeast are in iOS SDK 13, so would require Xcode 11 beta. I think you are getting the error because you are using an earlier Xcode version like Xcode 10.3 etc."

I was able to build with a new xcode version (11+)
